.github/workflows/.stylelintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+    "extends": ["stylelint-config-standard"],
+    "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+    "rules": {
+      "at-rule-no-unknown": null,
+      "scss/at-rule-no-unknown": [
+        true,
+        {
+          "ignoreAtRules": [
+            "tailwind",
+            "apply",
+            "variants",
+            "responsive",
+            "screen"
+          ]
+        }
+      ]
+    },
+    "csstree/validator": true,
+    "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css"]
+  }
```

Comment:
- [ ] Great work setting up your linter for css. 💯 However, please kindly ensure to extract your `.stylelintrc.json` file from your `.github/workflows` folder.🤓 This file should be located in the `root` directory of your project. 🔧
>Note - leave your `.linters.yml` file inside.